### PR TITLE
Added custom Solution

### PR DESCRIPTION
**Pull request by tgoyal63**

### CODE REVIEW

**Review by Teoriya — APPROVED** [review]
ek baar coments dekh kr merge krdena bhai , even agar kuch changes krne honge to abhi todo list me dalke isko merge krdena

**Comment by Teoriya** [comment]
https://www.notion.so/Backend-todo-e4467e4438144c8b8cc1896a0555b547?pvs=4 

yaha kaam add krdena jo chahiye

File: src/customSolutions/customIntegrations/tagMango/models/tmMapper.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+		},
+		tmCredential: {
+			type: mongoose.Schema.Types.ObjectId,
+			ref: "tmCredential",
+		},
+		customer: {
+			type: mongoose.Schema.Types.ObjectId,
+			ref: "customer",
+		},
+		metadata: { type: Object },
```
[the remark this answers is not in the document]
Comment (tgoyal63) [reply src/customSolutions/customIntegrations/tagMango/models/tmMapper.ts:33]: Metadata related to tagMango

File: src/models/mongoDB/service.model.ts [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
-			ref: "spreadsheet",
-			unique: true,
-			sparse: true,
-		},
+		// spreadsheet: {
+		// 	type: mongoose.Schema.Types.ObjectId,
+		// 	ref: "spreadsheet",
+		// 	unique: true,
+		// 	sparse: true,
+		// }, // already in spreadsheet model, not required here
```
[the remark this answers is not in the document]
Comment (tgoyal63) [reply src/models/mongoDB/service.model.ts:24]: Already fixed it.

**Pull request merged**